roadmap.md

```diff
@@ -19,28 +18,30 @@ Technology Roadmaps
 
 Architecture:
 
-- [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md)
+- [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md)
 - [Native Compilation](https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md)
-- [.NET Core Tools/CLI](https://github.com/dotnet/cli/blob/master/Documentation/intro-to-cli.md)
+- [.NET Core Tools/CLI](https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/intro-to-cli.md)
 
 Active ports:
 
-- Languages: F# (C# and VB are done).
-- OSes: Linux, OS X, FreeBSD and Windows.
-- Linux distros: RHEL, Debian family, CentOS.
+- Languages: C#, VB, and F#.
+- OS: Linux, macOS, FreeBSD and Windows.
```

Comment:
We should validate how active the FreeBSD port is at this point. I honestly don't know.